ibm/service/vpc/data_source_ibm_is_backup_policy_plan_test.go

```diff
@@ -43,6 +43,7 @@ func TestAccIBMIsBackupPolicyPlanDataSourceBasic(t *testing.T) {
 					resource.TestCheckResourceAttrSet("data.ibm_is_backup_policy_plan.is_backup_policy_plan", "lifecycle_state"),
 					resource.TestCheckResourceAttrSet("data.ibm_is_backup_policy_plan.is_backup_policy_plan", "name"),
 					resource.TestCheckResourceAttrSet("data.ibm_is_backup_policy_plan.is_backup_policy_plan", "resource_type"),
+					resource.TestCheckResourceAttrSet("data.ibm_is_backup_policy_plan.is_backup_policy_plan", "remote_copy_policies.#"),
```

Comment:
add test case to call TestAccIBMIsBackupPolicyRemoteCopyPolicies and then verify all the fields here